paperqa/agents/task.py

```diff
@@ -487,3 +493,229 @@ def __len__(self) -> int:
     LitQAv2TaskDataset.__module__,
     LitQAv2TaskDataset.__name__,
 )
+
+
+class LFRQAPairwiseEvalEnv(GradablePaperQAEnvironment):
```

Comment:
Can you add a docstring for the class? Just at least document what LFR stands for